modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,54 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix
+:pluginname: Merge Tags
+:plugincode: mergetags
+:pluginminimumplan: tiertwo
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field) from a drop-down list when a specified prefix is typed.
+
+Once the merge tag is inserted, the plugin leaves
+
+* a non-editable variable wrapped in a prefix; and
+* a suffix making the tag easily identifiable.
+
+The {pluginname} plugin provides an xref:autocompleter.adoc[autocompleter] for adding a merge tag without using the toolbar button or menu item.
+
+The autocompleter is triggered by typing a specified prefix, controlled by the xref:#mergetags_prefix[`+mergetags_prefix+`] option.
+
+By default, this prefix is `{{`.
+
+Entering characters after the prefix begins filtering the merge tags list.
```

Comment:
This looks like it should be in a "basic setup" heading instead of an introduction. See the footnotes docs for an example: https://github.com/tinymce/tinymce-docs/pull/2496/files#diff-aa7b214001248b6db84663e04401e95c3b8ddaf24c582e47842a00da0d48f86eR25

Reply:
To quote the commit message:

> Multiple explanatory paragraphs moved from opening (intro) section to immediately following the Basic setup code sample.